src/VisualStudio/Core/Def/Implementation/Remote/RemoteHostOptions.cs

```diff
@@ -28,13 +26,12 @@ internal static class RemoteHostOptions
             storageLocations: new LocalUserProfileStorageLocation(InternalFeatureOnOffOptions.LocalRegistryPath + nameof(SolutionChecksumMonitorBackOffTimeSpanInMS)));
 
         // use 64bit OOP
-        public static readonly Option<bool> OOP64Bit = new Option<bool>(
-            nameof(InternalFeatureOnOffOptions), nameof(OOP64Bit), defaultValue: false,
+        public static readonly Option2<bool> OOP64Bit = new Option2<bool>(
```

Comment:
>OOP64Bit [](start = 45, length = 8)

Do we need this option?

Reply:
This is the option exposed through the UI.

Reply:
Let's rename it to `RunOutOfProc` then.

---
In reply to: [468745130](https://github.com/dotnet/roslyn/pull/46578#discussion_r468745130) [](ancestors = 468745130)

Reply:
I prefer to keep the current name for easier identification of the relation between this and the UI option.